spec/support/factory_bot.rb

```diff
@@ -1,10 +1,10 @@
 require 'factory_bot'
 
 FactoryBot.define do
+  num = rand()
   factory :user do
-    sequence(:email) {|n| "user#{n}@example.com" }
+    sequence(:email) {|n| "user#{n}@#{num}.com" }
```

Comment:
factory bot will auto increment the `n` value for us, we don't need to use `rand`! (which could technically give us the same number twice)